example/example.html

```diff
@@ -4,8 +4,8 @@
   <meta charset="UTF-8">
   <title>Editor.js 🤩🧦🤨 example</title>
   <link href="https://fonts.googleapis.com/css?family=PT+Mono" rel="stylesheet">
-  <link href="assets/demo.css" rel="stylesheet">
-  <script src="assets/json-preview.js"></script>
+  <link href="/assets/demo.css" rel="stylesheet">
```

Comment:
tests should not affect the example pages

Reply:
done. All changes in example are reverted